pkg/ldd/ldd_unix.go

```diff
@@ -70,22 +70,34 @@ func follow(l string, names map[string]*FileInfo) error {
 	}
 }
 
-// runinterp runs the interpreter with the --list switch
-// and the file as an argument. For each returned line
-// it looks for => as the second field, indicating a
-// real .so (as opposed to the .vdso or a string like
-// 'not a dynamic executable'.
+func LdSo() (string, error) {
```

Comment:
the rule is if it starts with a capital you have to write a description. 

// LdSo returns the path of a runtime linker/loader.